experimenter/experimenter/changelog/templates/changelog-base.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
```

Comment:
here since it is already in changelog/templates/, you can omit `changelog` prefix, and just keep `base.html`

Reply:
so technically you can add a` template` folder inside django root project and each subfolder for the app in your case `changelog`, I am not sure which base.html you are referring to, but I am referring to have this structure
```general 
project_name/
    manage.py
    project_name/
        __init__.py
        settings.py
        urls.py
        ...
    templates/
        base.html
        app1/
            template1.html
            template2.html
        app2/
            template3.html
            template4.html
    app1/
        __init__.py
        models.py
        views.py
        ...
    app2/
        __init__.py
        models.py
        views.py
        ...
    ...

Reply:
So I have updated the templates to be according to the above file structure. The only thing is that I named the base template as `experimenter_base.html` as the template structure that was being followed by the legacy ui earlier is having conflicts with the new `base.html` as the files are collected altogether in the template engine. The order in which the template dirs are set helps in using the name `base.html` but if done so few tests fail as those refer to a different `base.html` // @yashikakhurana